elyra/metadata/schemas/component-registry.json

```diff
@@ -38,11 +38,11 @@
           "title": "Category Names",
           "description": "Category names associated with this Component Registry (the components defined in this registry will be organized in the component palette according to these categories)",
           "type": "array",
-          "items": [
```

Comment:
FWIW, the `items` issue is also addressed in the BYO Schemas PR.  This started raising a validation error once the version of `jsonschema` went to `4.0.1` (from `3.2.0`) and must enforce more stringent validators as it supports an updated draft of the JSON schema.